server/src/main/java/org/opensearch/rest/BaseRestHandler.java

```diff
@@ -66,6 +66,8 @@
  */
 public abstract class BaseRestHandler implements RestHandler {
 
+    public static List<Route> routes;
```

Comment:
This shouldn't be public if it's meant to be consumed via `getRoutes()`. Should it be protected? Probably not either.